internal/mackerel/aws_integration.go

```diff
@@ -22,6 +22,25 @@ type AWSIntegrationModel struct {
 	IncludedTags types.String `tfsdk:"included_tags"`
 	ExcludedTags types.String `tfsdk:"excluded_tags"`
 
+	AWSIntegrationSerfvices
+}
+
+type AWSIntegrationDataSourceModel struct {
+	ID           types.String `tfsdk:"id"`
+	Name         types.String `tfsdk:"name"`
+	Memo         types.String `tfsdk:"memo"`
+	Key          types.String `tfsdk:"key"`
+	SecretKey    types.String `tfsdk:"-"`
```

Comment:
AWS Integration resources contain an AWS secret key in their state, whereas data sources do not.